src/assets/image/icons/connections/index.ts

```diff
@@ -30,3 +30,5 @@ export { default as ZoomIcon } from "@assets/image/icons/connections/Zoom.svg?re
 export { default as EmptyCircleIcon } from "@assets/image/icons/connections/EmptyCircle.svg?react";
 // Taken from https://www.svgrepo.com/svg/333141/linear
 export { default as LinearIcon } from "@assets/image/icons/connections/linear.svg?react";
+// Taken from https://www.svgrepo.com/svg/354306/salesforce
```

Comment:
The license in the link is under [Logo License](https://www.svgrepo.com/page/licensing/#Logo), which is less "free" than MIT.

I found the same logo under MIT license, let's use it :)
https://svgmix.com/item/zglqJw/salesforce